src/server/sql/weather_location/delete_weather_location.sql

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+DELETE 
```

Comment:
I'm noting that you cannot delete a location if there is any weather data since here is no way to remove those. I think we should discuss how to deal with this. Unlike readings you can recreate weather data so removing a location is not as serious. Thus, the readings can also be removed but this would require a cascade. However, when locations get tied to meters/groups then that is another constraint where I think the admin needs to be warned and remove them manually. This is how units, etc. work.